### PR TITLE
minikube: set version metadata to program.

### PR DESCRIPTION
**Pull request by gbrlsnchs**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/minikube/template [hunk 1]
```diff
@@ -24,6 +25,13 @@ pre_configure() {
 
 pre_build() {
 	make pkg/minikube/assets/assets.go pkg/minikube/translate/translations.go
+
+	# Set up correct commit information to be embedded in the CLI.
+	_tmpdir=$(mktemp -d)
+	git clone --depth 1 --branch v${version} https://github.com/kubernetes/minikube ${_tmpdir}
+	cd ${_tmpdir}
+	go_ldflags+=" -X k8s.io/minikube/pkg/version.gitCommitID=$(git rev-parse --short HEAD)"
+	rm -rf ${_tmpdir}
```
Comment (sgn) [inline srcpkgs/minikube/template:34]: I would simply set `-X k8s.io/minikube/pkg/version.gitCommitID=v${version}`

Comment (gbrlsnchs) [reply srcpkgs/minikube/template:34]: I'd rather have no commit information then. I mean, version and commit are meant to be different things in this case. Using `--depth 1` and `--branch` was the best way I found to accomplish fetching such information without impacting much in build time while not having to manually update the commit hash. Is it that bad? 😞

Comment (Gottox) [reply srcpkgs/minikube/template:34]: These informations are not mandatory for compiling and running.

**Comment by Gottox** [comment]
This is already fixed in #29303.

File: srcpkgs/minikube/template (hunk 1 above)
Comment (gbrlsnchs) [reply srcpkgs/minikube/template:34]: Sure, I just wanted to improve information in the binary distributed by Void, although my solution was not ideal.

**Comment by gbrlsnchs** [comment]
> This is already fixed in #29303.

Thanks, the solution provided is a good one.

**Pull request closed without merging**